press/press/doctype/site/site.py

```diff
@@ -1409,11 +1411,34 @@ def login_as_team(self, reason=None):
 		if self.additional_system_user_created:
 			team_user = frappe.db.get_value("Team", self.team, "user")
 			sid = self.get_login_sid(user=team_user)
-			return f"https://{self.host_name or self.name}/desk?sid={sid}"
+			if self.standby_for_product:
+				redirect_route = (
+					frappe.db.get_value("Product Trial", self.standby_for_product, "redirect_to_after_login")
+					or "/desk"
+				)
+			else:
+				redirect_route = "/desk"
+			return f"https://{self.host_name or self.name}{redirect_route}?sid={sid}"
 
 		frappe.throw("No additional system user created for this site")
 		return None
 
+	@site_action(["Active"])
+	def login_as_user(self, user_email, reason=None):
```

Comment:
anyone who has access to FC can login to the site?

Reply:
Yes, but before that, they'll have to verify as the user to log in. btw `@site_action` isn't a wrapper on `frappe.whitelist`.